hack/test

```diff
@@ -38,22 +38,24 @@ function fmt_pass {
 		fi
 	done
 
+	checkPkgs=$(go list ./client/... ./cmd/... ./pkg/... ./test/e2e/...)
+	
 	go get honnef.co/go/tools/cmd/gosimple || true
 	if which gosimple >/dev/null; then
 		echo "Checking gosimple..."
-		checkRes=$(gosimple $(go list ./client/... ./cmd/... ./pkg/... ./test/e2e/...)) || true
-			if [ -n "${checkRes}" ]; then
-				echo -e "gosimple checking failed:\n${checkRes}"
-				exit 255
-			fi
+		checkRes=$(gosimple $checkPkgs)
```

Comment:
I can double check this quick but from what I recall when the `gosimple` check finds some improvements in the code it stopped the entire script since we have `set -o errexit` at the top. So the `|| true` was there to continue despite finding errors so we could display them.